yt_dlp/extractor/dropout.py

```diff
@@ -0,0 +1,104 @@
+        'info_dict': {
+            'id': '1915774',
+            'display_id': 'misfits-magic-holiday-special',
+            'ext': 'mp4',
+            'title': 'Misfits & Magic Holiday Special',
+            'description': 'The magical misfits spend Christmas break at Gowpenny, with an unwelcome visitor.',
+            'release_date': '20211215',
+            'thumbnail': 'https://vhx.imgix.net/chuncensoredstaging/assets/d91ea8a6-b250-42ed-907e-b30fb1c65176-8e24b8e5.jpg',
+            'duration': 11698,
+            'uploader_id': 'user80538407',
+            'uploader_url': 'https://vimeo.com/user80538407',
+            'uploader': 'OTT Videos'
+        },
+        'expected_warnings': ['Ignoring subtitle tracks found in the HLS manifest']
+    }]
+
+    def _get_authenticity_token(self):
+        signin_page = self._download_webpage(self._LOGIN_URL, video_id=self.display_id,
+                                             note='Getting authenticity token')
+        authenticity_token = self._html_search_regex(
+            r'name=["\']authenticity_token["\'] value=["\'](.+?)["\']',
+            signin_page, 'authenticity_token')
+        return authenticity_token
+
+    def _login(self):
+        # Get & check username and password
+        username, password = self._get_login_info()
+        if not (username and password):
+            self.raise_login_required()
+
+        # Make POST login request
+        payload = {
+            'email': username,
+            'password': password,
+            'authenticity_token': self._get_authenticity_token(),
+            'utf8': True
+        }
+        response = self._download_webpage(self._LOGIN_URL, video_id=self.display_id,
+                                          note='Logging in', data=urlencode_postdata(payload))
+
+        # Make sure login was successful
+        user_has_subscription = self._html_search_regex(r'user_has_subscription: ["\'](.+?)["\']',
+                                                        response, 'success')
+        if str(user_has_subscription).lower() == 'true':
+            return response
+        else:
+            self._logout()
+            self.raise_login_required('Incorrect username/password, or account is not subscribed')
+
+    def _logout(self):
+        self._download_webpage(self._LOGOUT_URL, self.display_id, note='Logging out')
+
+    def _real_extract(self, url):
+        self.display_id = self._match_id(url)
+
+        self._login()
+
+        # Get webpage and embed_url, and log out
+        webpage = self._download_webpage(url, self.display_id, note='Downloading video webpage')
+        self._logout()
+        embed_url = self._html_search_regex(r'embed_url: ["\'](.+?)["\']', webpage, 'url')
+
+        # More metadata
+        id = self._html_search_regex(r'embed.vhx.tv/videos/(.+?)\?', embed_url, 'id')
+        title = self._html_search_regex(r'<title>(.+?)</title>', webpage, 'title')
+        if title:
+            title = title.split(' - ')[0]
```

Comment:
```suggestion
        title = self._html_search_regex(r'<title>(.+?)</title>', webpage, 'title').split(' - ')[0]
```
since title is mandatory.

But are you sure the actual title can never have a ` - ` in it? `remove_end` may be more appropriate